R/scholar.r

```diff
@@ -252,3 +252,86 @@ plot_coauthors <- function(network, size_labels = 5) {
     ggplot2::labs(title = paste0("Network of coauthorship of ", network$author[1])) +
     ggraph::theme_graph(title_size = 16, base_family = "sans")
 }
+
+
+
+
+
+
+
+
+##' Get author order.
+##'
+##' Get author rank in authors list.
+##'
+##' @examples
+##' library(scholar)
+##'
+##' id <- "bg0BZ-QAAAAJ&hl"
+##'
+##' publications <- scholar::get_publications(id)
+##' publications <- publications$author
+##' author <- scholar::get_profile(id)
+##' author <- author$name
+##'
+##' author_position(publications, author)
+##'
+##' @param publications list of publication authors
+##' @param author author's name to look for
+##'
+##' @return dataframe with author's position and position percentage.
```

Comment:
I think the `position percentage` is not very intuitive, so you might want to describe what it is.

Reply:
Renamed percentage to "Position_Normalized" and gave details in description